layouts/side-bar-menu/index.tsx

```
@@ -39,55 +46,102 @@ const SIDE_BAR_MENU_BOTTOM = [
+            borderTopColor: "lightgray",
+            borderTopWidth: 1,
+            paddingBottom: 16,
+            paddingTop: 16,
+          }}
+        >
+          {SIDE_BAR_MENU_BOTTOM.map((item) => (
+            <Pressable
+              onPress={() => {
+                router.push(item.href as Href);
+              }}
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)